src/discussions/posts/post/PostLink.jsx

```diff
@@ -31,12 +34,9 @@ function PostLink({
       className="discussion-post list-group-item list-group-item-action p-0 text-decoration-none text-gray-900"
       to={linkUrl}
     >
-      {post.abuseFlagged && (
-        <div className="align-items-center bg-danger-100 d-flex flex-fill p-1">
-          <Icon className="text-danger-700" src={Flag} />
-          <span className="text-gray-700 x-small">{intl.formatMessage(messages.contentReported)}</span>
-        </div>
-      )}
+      { post.closed && BANNER_TYPE.closed(intl.formatMessage(messages.closed))}
+      { post.abuseFlagged && BANNER_TYPE.reported(intl.formatMessage(messages.contentReported)) }
+      { post.pinned && BANNER_TYPE.pinned(intl.formatMessage(messages.pinned)) }
```

Comment:
I don't think you need to add the post closed banner here. 
Also move these banners inside the Post component and you won't need to repeat them twice, just adding them to the Post component will make them show up both on the left and right side when the post is rendered.